scrape_linkedin/Scraper.py

```diff
@@ -85,6 +85,7 @@ def scroll_to_bottom(self):
             'button[aria-expanded="false"].pv-skills-section__additional-skills',
             'button[aria-expanded="false"].pv-profile-section__see-more-inline',
             'button[aria-expanded="false"].pv-top-card-section__summary-toggle-button',
+            'button[aria-selected="false"].artdeco-tab',
```

Comment:
Actually, I'm not sure because if we don't do this, there'll be only 2 given recommendations. So "See more" must be clicked, if we don't click on this, then it is not rendered. I agree about the infinite loop, but it seems like otherwise, we need to add some extra code to limit clicks on this element

Reply:
So I added a new commit. I wonder how to make "scroll_to_bottom" less messy. Do you have any ideas? I tried different ways, but this one at least works.